sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/AmqpChannelProcessor.java

```diff
@@ -63,28 +63,6 @@ public class AmqpChannelProcessor<T> extends Mono<T> implements Processor<T, T>,
     private volatile Disposable connectionSubscription;
     private volatile Disposable retrySubscription;
 
-    /**
-     * Creates an instance of {@link AmqpChannelProcessor}.
-     *
-     * @param fullyQualifiedNamespace The fully qualified namespace for the AMQP connection.
-     * @param entityPath The entity path for the AMQP connection.
-     * @param endpointStatesFunction The function that returns the endpoint states for the AMQP connection.
-     * @param retryPolicy The retry policy for the AMQP connection.
-     * @param logger The logger to use for this processor.
-     * @deprecated Use constructor overload that does not take {@link ClientLogger}
-     */
-    @Deprecated
-    public AmqpChannelProcessor(String fullyQualifiedNamespace, String entityPath,
-        Function<T, Flux<AmqpEndpointState>> endpointStatesFunction, AmqpRetryPolicy retryPolicy, ClientLogger logger) {
-        this.endpointStatesFunction
-            = Objects.requireNonNull(endpointStatesFunction, "'endpointStates' cannot be null.");
-        this.retryPolicy = Objects.requireNonNull(retryPolicy, "'retryPolicy' cannot be null.");
-        Map<String, Object> loggingContext = new HashMap<>(1);
-        loggingContext.put(ENTITY_PATH_KEY, Objects.requireNonNull(entityPath, "'entityPath' cannot be null."));
-        this.logger = new ClientLogger(getClass(), loggingContext);
-        this.errorContext = new AmqpErrorContext(fullyQualifiedNamespace);
-    }
```

Comment:
thanks for getting rid of all these unused Ctrs :)

Reply:
apparently not all of them are unused 🤐